app/webpacker/components/Panel/SeniorDelegate/Regions/Subregion.jsx

```diff
@@ -29,7 +29,7 @@ const isLead = (role) => role.metadata.status === 'leader';
 
 const canPromote = (role) => (
   [
-    statusObjectOfGroupType(role.group.group_type).trainee_delegate,
+    statusObjectOfGroupType(role.group.group_type).candidate_delegate,
```

Comment:
Are you sure you changed the correct line in this file?

Reply:
(Although this is irrelevant if we revert the "junior -> candidate" change

Reply:
Oops my bad, reverted. Thanks.